idc.Makefile

```diff
@@ -96,5 +98,47 @@ up:  download-default-certs docker-compose.yml start
 .SILENT: start
 start:
 	docker-compose up -d
-	sleep 5
-	docker-compose exec drupal /bin/sh -c "while true ; do echo \"Waiting for Drupal to start ...\" ; if [ -d \"/var/run/s6/services/nginx\" ] ; then s6-svwait -u /var/run/s6/services/nginx && exit 0 ; else sleep 5 ; fi done"
+	$(MAKE) wait-for-drupal
+
+.PHONY: wait-for-drupal
```

Comment:
Just curious, is this refactoring of a `wait-for-drupal` target germane to the static build?  If not, maybe a separate PR for this?

Reply:
In an earlier revision, it was needed.  I can take it out and add a separate PR at some point

Reply:
I noticed that when reverting the `wait-for-drupal` change, I needed to add back in the  `-T` part of the previous change, as it was necessary for running in github actions (which doesn't have a tty)